tests/unit/lib/foxbox/event-dispatcher_test.js

```diff
@@ -0,0 +1,423 @@
+import EventDispatcher from 'js/lib/foxbox/event-dispatcher';
+
+describe('EventDispatcher >', function () {
```

Comment:
question: Do we have a good reason to use `function () {` as opposed to `() => {` in the test files?